packages/toolkit/src/view/hub/hub-view/LatestChangesCard.tsx

```diff
@@ -37,11 +37,12 @@ export const LatestChangesCard = () => {
   }
 
   return (
-    <div className="mt-4 flex flex-col gap-y-4 rounded-sm border border-semantic-bg-line p-4">
+    <div className="mt-4 flex flex-col gap-y-4 rounded-sm border border-semantic-bg-line p-4 bg-semantic-bg-primary">
```

Comment:
@thewbuk I will suggest that you use "mb-4" on the upper component instead of using mt-4 or gap-y-4. We want to unify the direction of margin to avoid margin collapse issue

Reply:
Please unify this behavior across all the hub related components